src/inc/public/AppxPackaging.hpp

```diff
@@ -124,6 +125,7 @@ interface IAppxManifestQualifiedResourcesEnumerator;
 interface IAppxManifestQualifiedResource;
 interface IAppxBundleFactory;
 interface IAppxBundleWriter;
+interface IAppxBundleWriter4;

```

Comment:
>IAppxBundleWriter4 [](start = 10, length = 18)

You also need to define the interface for non windows platforms (that's why the build is failing in non windows). Look a AppxPackaging.h which gets generated by AppxPackaging.idl and mimic how IAppxBundleWriter4 is defined in here.

Reply:
I defined interface for IAppxBundleWriter4.